met-web/public/index.html

```diff
@@ -1,26 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <title>Modern Engagement Tool</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site of MET"
-    />
-    <script src="%PUBLIC_URL%/config/config.js"></script>
-    <script src="/snowplow.js" charset="UTF-8"></script>
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <link
-      rel="stylesheet"
-      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
-    />
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-  </body>
+    <head>
+        <meta charset="utf-8" />
+        <title>Modern Engagement Tool</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="theme-color" content="#000000" />
+        <meta name="description" content="Web site of MET" />
+        <script src="%PUBLIC_URL%/config/config.js"></script>
+        <script src="/snowplow.js" charset="UTF-8"></script>
+        <link rel="icon" href="%PUBLIC_URL%/BCFavIcon.png" />
```

Comment:
I think it's better to have a favicon.ico file if it's easy. All modern browsers support .png, but maybe some public users are operating on weird browsers. And the name should be "favicon"